store/index.js

```diff
@@ -135,6 +135,11 @@ export default function configureStore(initialState) {
                         });
                     }
                 });
+            }).catch((error) => {
+                store.dispatch({
+                    type: General.STORE_REHYDRATION_FAILED,
```

Comment:
I've got a pending PR to add this constant to mattermost-redux... if we think we need it. No one is going to handle it right now. Feedback requested!

Reply:
I'm not sure if I prefer simply a log message for now. Anyway, I think it must be in the webapp redux flow, not in the mattermost-redux one. Because all the storage thing is handle independently from RN app and webapp.

Reply:
I'll move the constant locally -- the log message would defeat the point of this change, at present ;P

Reply:
hahaha, you are completely right, I'm an idiot :)